carAPI/src/main/java/com/everis/boundary/CarResource.java

```diff
@@ -23,35 +20,27 @@
 import javax.ws.rs.core.Response.Status;
 import javax.ws.rs.core.UriInfo;
 
+import com.everis.control.CarService;
 import com.everis.entity.Car;
 
 @Path("/cars")
 @Produces(MediaType.APPLICATION_JSON)
 @Consumes(MediaType.APPLICATION_JSON)
 public class CarResource {
 	
-	private static Map<Long, Car> cars = new HashMap<Long, Car>();
+	private CarService carService = new CarService();
 	
-	public CarResource() {
-		// TODO Auto-generated constructor stub
-		Car newCar1 = new Car(1, "Coche1", LocalDateTime.now(), "Espa�a", LocalDateTime.now(), LocalDateTime.now());
-		Car newCar2 = new Car(2, "Coche2", LocalDateTime.now(), "Espa�a", LocalDateTime.now(), LocalDateTime.now());
-		Car newCar3 = new Car(3, "Coche3", LocalDateTime.now(), "Espa�a", LocalDateTime.now(), LocalDateTime.now());
-		cars.put(1l, newCar1);
-		cars.put(2l, newCar2);
-		cars.put(3l, newCar3);
-	}
 	//Devuelve todos los coches
 	@GET
 	public List<Car> getCars(){
-		return new ArrayList<Car>(cars.values());
+		return carService.getCars();
```

Comment:
Please wrap the car list in a Response this way we keep a consistent coding style